### PR TITLE
Fix boat entity (remove MediaWithTheme Mecanics)

### PR DESCRIPTION
**Pull request by MaximeWrlz**

### CODE REVIEW

**Comment by FlavianEng** [comment]
Task linked: [CU-3c0tr1r Fix Boat entity](https://app.clickup.com/t/3c0tr1r)

**Review by ManonFargues — DISMISSED** [review]

File: bin/fixtures-load.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
+#!/bin/bash
+
 fixtures-ts-node-commonjs load \
         ./src/fixtures/media.yml \
```
Comment (FlavianEng) [inline bin/fixtures-load.sh:4]: Don't we remove media too ?

File: src/domains/character/character.entity.ts [hunk 2]
```diff
@@ -22,6 +22,5 @@ export default class Character {
   public media!: number;
```
Comment (FlavianEng) [inline src/domains/character/character.entity.ts:22]: Don't we remove this field ?

File: src/modules/domain.module.ts [hunk 3]
```diff
@@ -19,9 +19,6 @@ import LevelService from '@level/level.service';
 import MediaController from '@media/media.controller';
 import MediaRepository from '@media/media.repository';
 import MediaService from '@media/media.service';
```
Comment (FlavianEng) [inline src/modules/domain.module.ts:21]: Don't we remove this entity ?

File: src/shared/entities.export.ts [hunk 4]
```diff
@@ -3,7 +3,6 @@ import Character from '@character/character.entity';
 import Game from '@game/game.entity';
 import Level from '@level/level.entity';
 import Media from '@media/media.entity';
```
Comment (FlavianEng) [inline src/shared/entities.export.ts:5]: Don't we remove this entity ?

File: tsconfig.json [hunk 5]
```diff
@@ -33,7 +33,6 @@
       "@interceptors/*": ["src/interceptors/*"],
       "@interfaces/*": ["src/interfaces/*"],
       "@level/*": ["src/domains/level/*"],
-      "@media-with-theme/*": ["src/domains/media-with-theme/*"],
       "@media/*": ["src/domains/media/*"],
```
Comment (FlavianEng) [inline tsconfig.json:36]: Don't we remove media ?

**Review by FlavianEng — CHANGES_REQUESTED** [review]

**Review by FlavianEng — APPROVED** [review]

**Review by DrakenLine — APPROVED** [review]

**Pull request merged**